.docker/php/80-fpm-prod/Dockerfile

```diff
@@ -1,38 +1,36 @@
 FROM php:8.0-fpm
 
 LABEL description="Fogos.pt API Production PHP 8.0 image"
+LABEL org.opencontainers.image.source https://github.com/FogosPT/fogosapi
 
-RUN apt-get update
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
-RUN apt-get install -y --no-install-recommends locales
-RUN cp /usr/share/i18n/SUPPORTED /etc/locale.gen && locale-gen
-
-RUN curl -sL https://deb.nodesource.com/setup_12.x | bash -
-
-RUN apt-get install -y --no-install-recommends \
+RUN apt-get update && \
```

Comment:
I'm not very comfortable in merging all of this. The `locale-gen` command takes a lot of time and, as far as I know, if we modify this line (by adding or removing a package, for example) the layer cache will be busted and the entire work will need to be done, again. Is this really necessary?

Reply:
Ok makes sense, I can split in 2 RUN commands the idea is run the commands ` apt-get clean && rm -rf /var/lib/apt/lists/*` at the end to avoid trash in the layers

Reply:
Sure!